app/templates/view_users.html

```diff
@@ -23,87 +21,71 @@
 {% block mainContent %}
   <h1 class="govuk-heading-xl">Find a user</h1>
 
-  <form action="{{ url_for('.find_user_by_email_address') }}" method="get" class="question">
-    {%
-      with
-      question = "Find a user by email",
-      name = "email_address"
-    %}
-      {% include "toolkit/forms/textbox.html" %}
-    {% endwith %}
+  <form action="{{ url_for('.find_user_by_email_address') }}" method="get" class="question" novalidate>
+    {{ govukInput({
+      "label": {
+        "text": "Find a user by email",
+        "classes": "govuk-label--m",
+        "isPageHeading": false
```

Comment:
This isn't a required field and `false` is the default, but no harm in spelling it out.

Reply:
Removed.